FreeRTOS/FreeRTOS
# Add Github Actions builds for FreeRTOS+ WinSim Demos

Add Github Actions builds for FreeRTOS+ Demos

Description
-----------
Update the various FreeRTOS Plus Windows Simulator Demos to target Visual Studio 2019. Build them with github actions.

Test Steps
-----------
Observe that the builds pass.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
